.example.env 

```diff
@@ -0,0 +1,4 @@
+POSTGRES_DB=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+DATABASE_URL="postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:5555/${POSTGRES_DB}?schema=public"
```

Comment:
hmm, why custom port (5555)?

Reply:
I have another postgress container running with strapi and I don't want to kill it :p